config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml

```diff
@@ -179,3 +179,48 @@ presets:
         path: ca.crt
       - key: vmc-vpn-tls.key
         path: tls.key
+- labels:
+    # For kubernetes-sigs/cluster-api-provider-vsphere janitor
+    preset-vsphere-janitor-config: "true"
+  env:
+  - name: BOSKOS_HOST
+    value: http://192.168.6.138:32222
+  - name: GOVC_URL
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-url
+  - name: GOVC_USERNAME
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-cluster-api-provider-vsphere-user
+  - name: GOVC_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-cluster-api-provider-vsphere-password
+  - name: VSPHERE_TLS_THUMBPRINT
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-thumbprint
+  volumeMounts:
+  - name: vmc-vpn
+    mountPath: /root/.openvpn
+  volumes:
+  - name: vmc-vpn
+    secret:
+      secretName: cluster-api-provider-vsphere-ci
+      defaultMode: 256
+      items:
+      - key: vmc-vpn-config.ovpn
+        path: prow.ovpn
+      - key: vmc-vpn-client.crt
+        path: client.crt
+      - key: vmc-vpn-client.key
+        path: client.key
+      - key: vmc-vpn-ca.crt
+        path: ca.crt
+      - key: vmc-vpn-tls.key
```

Comment:
Wondering if we should keep `vmc-capv-services-kubeconfig` for now. Otherwise the janitor is still broken after my current PR merges (IPAM cleanup is a later cleanup step)

(same for the corresponding volumeMount above)

Reply:
(totally fine for me if you want to keep this PR on hold until then, shouldn't take very long)